test/test_client.py

```diff
@@ -217,10 +217,7 @@ def test_getattr(self):
 
     def test_iteration(self):
         client = self.client
-        if "PyPy" in sys.version:
-            msg = "'NoneType' object is not callable"
-        else:
-            msg = "'MongoClient' object is not iterable"
+        msg = "'MongoClient' object is not iterable"
```

Comment:
Won't this now fail on the old pypy versions in EVG?

Reply:
Oh ha, landmines everywhere.  😅

Reply:
Fixed, and scheduled some pypy tasks